### ISSUE TITLE
Xamarin Forms Platform Specific XAML Error

### ISSUE THREAD

**Issue by vsfeedback** [xamarin/Xamarin.Forms#10832]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1042599/xamarin-forms-platform-specific-xaml-error.html)._

---
I tried to use platform specific (in this case android's tabbed page). It shows error (last 3 properties) on the editor and no intellisense were provided for those 3 properties, though it worked when i run the application. Please fix this so it wont be error on the editor and shows in vs intellisense. Thanks.

```
xmlns:android="clr-namespace:Xamarin.Forms.PlatformConfiguration.AndroidSpecific;assembly=Xamarin.Forms.Core"
android:TabbedPage.IsSwipePagingEnabled="False"
android:TabbedPage.IsSmoothScrollEnabled="False"
android:TabbedPage.ToolbarPlacement="Bottom"
android:TabbedPage.BarItemColor="{StaticResource AccentText2}"
android:TabbedPage.BarSelectedItemColor="{StaticResource AccentText}">
```

---
#### Visual Studio Feedback System on 5/21/2020, 02:17 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by samhouts** [comment]
Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by realivanjx** [comment]
[tabissuexam.zip](https://github.com/xamarin/Xamarin.Forms/files/4685623/tabissuexam.zip)

And this is what i am seeing now in xaml:

![xaml](https://user-images.githubusercontent.com/12814796/82961429-912b0980-9fe7-11ea-9469-f5c6607b80d1.png)

And almost same thing happened in code behind too:

![cs](https://user-images.githubusercontent.com/12814796/82961441-9e47f880-9fe7-11ea-9132-74176fda504b.png)

All those produce expected result and runs good on both android emulator and real device.

**Comment by jsuarezruiz** [comment]
Can reproduce the issue.
<img width="779" alt="Captura de pantalla 2020-06-01 a las 14 02 54" src="https://user-images.githubusercontent.com/6755973/83407264-9eddf600-a410-11ea-97ee-4d337317a019.png">

**Comment by realivanjx** [comment]
@jsuarezruiz Can you also reproduce it in the C# side?

**Comment by Kapusch** [comment]
Same here in my XAML file, but no intellisense error in code-behind (and it compiles) :
![image](https://user-images.githubusercontent.com/9576733/83567956-b907f900-a522-11ea-8ef6-ddb79f9c111d.png)

I'm using XF 4.6.0.726

**Comment by realivanjx** [comment]
@Kapusch Try to use `SetBarItemColor` and `SetSelectedBarItemColor` because `SetToolbarPlacement` in code behind is good i think.

**Comment by Kapusch** [comment]
Sounds good @realivanjx , except I am informed about using obsolete methods for both :

![image](https://user-images.githubusercontent.com/9576733/83684812-f2ef0300-a5e7-11ea-8454-9385dea38f1c.png)

**Comment by realivanjx** [comment]
Yes @Kapusch I also got warning on those 2 also not showing up in intellisense.
